### PR TITLE
💄 Update getApp page UI

### PR DESCRIPTION
**Pull request by shellyYG**

### CODE REVIEW

File: src/assets/images/app-logo.svg [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="187.683" height="52" viewBox="0 0 187.683 52">
+<svg xmlns="http://www.w3.org/2000/svg" width="160" height="52" viewBox="0 0 187.683 52">
```
Comment (nwingt) [inline src/assets/images/app-logo.svg:1]: Should override from Vue template

File: src/locales/en.json [hunk 2]
```diff
@@ -318,12 +318,12 @@
...
-      "Fact Check News",
-      "Public Discussion",
-      "Blogging Platform",
-      "Individual Websites"
+      {"mainVP": "The most user-friendly LikeCoin wallet",
+       "subVP": "The easiest way to send and receive LikeCoin." },
+      {"mainVP": "Check the likes statistics",
+      "subVP": "View the weekly, daily statistics of your LikeCoin bonus, down to each workpiece." },
+      {"mainVP": "LikeCoin staking",
+        "subVP": "Stake your LikeCoin to gain extra reward, and participate in community governance." }
```
Comment (nwingt) [inline src/locales/en.json:326]: I prefer not use nested key 😉

File: src/components/GetAppPage/GetAppPage.vue [hunk 3]
```diff
@@ -90,6 +96,11 @@ export default {
       default: '',
     },
   },
+  data() {
+    return {
+      singleFeature: '<div>feature.mainVP</div>',
```
Comment (nwingt) [inline src/components/GetAppPage/GetAppPage.vue:101]: What is this for?

File: src/components/GetAppPage/GetAppPage.vue [hunk 4]
```diff
@@ -1,78 +1,84 @@
...
+    <section class="container">
+      <hr class="separate-line">
+    </section>
+    <section class="footer-container">
+      <h1 class="slogan center">
+        <template v-if="$i18n.locale === 'zh-Hant'">
+          <div class="slogan-text center"> 立即下載 </div>
+        </template>
+        <template v-else>
+          <div class="slogan-text center"> Download Now </div>
```
Comment (nwingt) [inline src/components/GetAppPage/GetAppPage.vue:49]: Why don't use localization here?

**Review by nwingt — CHANGES_REQUESTED** [review]

File: src/components/GetAppPage/GetAppPage.vue (hunk 3 above)
Comment (shellyYG) [reply src/components/GetAppPage/GetAppPage.vue:101]: actually no use... will delete. :P

File: src/components/GetAppPage/GetAppPage.vue (hunk 4 above)
Comment (shellyYG) [reply src/components/GetAppPage/GetAppPage.vue:49]: because copied from legacy code lol 
will change it

File: src/assets/images/app-logo.svg [hunk 5]
```diff
@@ -1,4 +1,4 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="160" height="52" viewBox="0 0 187.683 52">
+<svg xmlns="http://www.w3.org/2000/svg" width="width" height="52" viewBox="0 0 187.683 52">
```
Comment (nwingt) [inline src/assets/images/app-logo.svg:1]: You can keep 160, it will be overwritten

File: src/components/GetAppPage/GetAppPage.vue [hunk 6]
```diff
@@ -25,13 +19,19 @@
 
     <section class="feature">
       <div>
-        <div v-for="(feature, key) in $t('GetAppPage.Feature')" :key="key" class="container side-by-side">
+        <div v-for="number in valueProps" :key="number" class="container side-by-side">
```
Comment (nwingt) [inline src/components/GetAppPage/GetAppPage.vue:22]: Vue can do this~
```suggestion
        <div v-for="number in 3" :key="number" class="container side-by-side">
```

File: src/components/GetAppPage/GetAppPage.vue [hunk 7]
```diff
@@ -4,19 +4,13 @@
     <section class="container top-container">
       <img class="feature-image" src="./getapp-banner.png">
       <img v-if="showCross" class="cross" src="./cross_3x.png">
-      
+
       <div class="center app-logo">
-        <AppLogo />
+        <AppLogo :width="width" />
```
Comment (nwingt) [inline src/components/GetAppPage/GetAppPage.vue:9]:
```suggestion
        <AppLogo width="160px" />
```

File: src/locales/en.json [hunk 8]
```diff
@@ -316,16 +316,17 @@
...
   },
+  "GetAppPage.Download": "Download Liker Land",
+  "GetAppPage.Manage.like": "Manage your LikeCoin",
+  "GetAppPage.Download.now": "Download Now",
+  "GetAppPage.Feature.mainVP1": "The most user-friendly LikeCoin wallet",
+  "GetAppPage.Feature.mainVP2": "Check the likes statistics",
+  "GetAppPage.Feature.mainVP3": "LikeCoin staking",
+  "GetAppPage.Feature.subVP1": "The easiest way to send and receive LikeCoin.",
+  "GetAppPage.Feature.subVP2": "View the weekly, daily statistics of your LikeCoin bonus, down to each workpiece.",
+  "GetAppPage.Feature.subVP3": "Stake your LikeCoin to gain extra reward, and participate in community governance.",
```
Comment (nwingt) [inline src/locales/en.json:329]: Can use array 
```js
{
  ...
  "GetAppPage.Feature.mainVP": [
     ...
  ],
  "GetAppPage.Feature.subVP": [
     ...
  ],
  ...
}
```

**Review by nwingt — CHANGES_REQUESTED** [review]

File: src/components/GetAppPage/GetAppPage.vue (hunk 6 above)
Comment (shellyYG) [reply src/components/GetAppPage/GetAppPage.vue:22]: wow! so lean

Comment (nwingt) [reply src/components/GetAppPage/GetAppPage.vue:22]: https://vuejs.org/v2/guide/list.html#v-for-with-a-Range

**Review by nwingt — APPROVED** [review]

**Pull request merged**